lxd/storage/drivers/driver_powerflex.go

```diff
@@ -204,23 +204,24 @@ func (d *powerflex) Validate(config map[string]string) error {
 		//  shortdesc: Whether to verify the PowerFlex Gateway's certificate
 		"powerflex.gateway.verify": validate.Optional(validate.IsBool),
 		// lxdmeta:generate(entities=storage-powerflex; group=pool-conf; key=powerflex.pool)
-		//
+		// If you want to specify the storage pool via its name, also set {config:option}`storage-powerflex-pool-conf:powerflex.domain`.
 		// ---
 		//  type: string
-		//  shortdesc: ID of the PowerFlex storage pool (if you want to specify the storage pool via its name, also set `powerflex.domain`)
+		//  shortdesc: ID of the PowerFlex storage pool
 		"powerflex.pool": validate.IsAny,
 		// lxdmeta:generate(entities=storage-powerflex; group=pool-conf; key=powerflex.domain)
-		//
+		// This option is required only if {config:option}`storage-powerflex-pool-conf:powerflex.pool` is specified using its name.
 		// ---
 		//  type: string
-		//  shortdesc: Name of the PowerFlex protection domain (only required if `powerflex.pool` is specified using its name)
+		//  shortdesc: Name of the PowerFlex protection domain
 		"powerflex.domain": validate.Optional(validate.IsAny),
 		// lxdmeta:generate(entities=storage-powerflex; group=pool-conf; key=powerflex.mode)
-		//
+		// The mode gets discovered automatically if the system provides the necessary kernel modules.
+                // Currently, only `nvme` is supported.
 		// ---
 		//  type: string
 		//  defaultdesc: the discovered mode
-		//  shortdesc: Gets discovered automatically if the system provides the necessary kernel modules; currently, only `nvme` is supported
+		//  shortdesc: Communication protocol
```

Comment:
Does this make sense? Or should it be "PowerFlex mode" - but that doesn't say much imo?

Reply:
@ru-fu powerflex.mode describes the method in which volumes are mapped to the local server as devices

Reply:
> But I think we should make clear that this option doesn't need to be provided since LXD will see that this mode is supported and will set it to `nvme` by default.

Sure, I moved that to the long description, so it's still there (just not in the "title").

Reply:
I see, sorry, missed it.